src/Pages/Host/Manual/SupportCard/styles.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+
+export const SupportCard = styled.div`
+  margin-top: 80px;
+  width: 340px;
+  height: 370px;
+  font-family: ${props => props.theme.fontContent};
```

Comment:
```suggestion
  font-family: ${({theme}) => theme.fontContent};
```
매개변수자리에서 바로 구조분해 가능합니다!